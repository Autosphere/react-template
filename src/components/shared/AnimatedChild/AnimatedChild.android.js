// @flow
import AnimatedChild from "./AnimatedChild.ios";

export default AnimatedChild;
