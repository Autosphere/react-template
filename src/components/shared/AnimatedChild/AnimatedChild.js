// @flow
type Props = {
  children: React$Element<*>
};

// No transition animations on web for now
const App = (props: Props) => props.children;

export default App;
