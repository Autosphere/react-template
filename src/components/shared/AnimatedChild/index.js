// @flow
import AnimatedChild from "./AnimatedChild";

export default AnimatedChild;
