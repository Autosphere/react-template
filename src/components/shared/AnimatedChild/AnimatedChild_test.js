// @flow

// Enzyme docs:
// http://airbnb.io/enzyme/docs/api/index.html

import React from "react";
import { shallow } from "enzyme";
import AnimatedChild from "./AnimatedChild";

const mockProps = {
  children: <div />
};

it("<AnimatedChild> renders the AnimatedChild content", () => {
  const wrapper = shallow(<AnimatedChild {...mockProps} />);

  expect(wrapper).toMatchSnapshot();
});
