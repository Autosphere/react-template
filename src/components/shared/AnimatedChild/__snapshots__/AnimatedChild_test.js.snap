// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AnimatedChild> renders the AnimatedChild content 1`] = `
<div
  className="template"
>
  http://www.example.com
</div>
`;
